### ISSUE TITLE
Weather need to train or not?

### ISSUE THREAD

**Issue by SmallLeishen** [yhgon/NoProp#1]
Hello, I have seen the code "loss.backward()" and "model.train()".

**Comment by azhuvath** [comment]
> Hello, I have seen the code "loss.backward()" and "model.train()".

Is it not backpropagating the error by calling loss.backward()?
